app/views/offers/offers_uk.scala.html

```diff
@@ -16,9 +16,8 @@ <h2 class="block__title">Digital</h2>
                     <ul class="block__list">
                         <li class="block__list-item">14-day free trial of our digital pack</li>
                         @UK.digitalPackSaving.map { saving => <li class="block__list-item"><strong>Save up to @saving% on the app store price</strong></li> }
-                        <li class="block__list-item">Access to the Daily Edition and the advert-free Guardian App Premium Tier</li>
+                        <li class="block__list-item">Access to the Daily Edition and the advert-free Guardian App Premium Tier available on Apple and Android</li>
```

Comment:
I think implies that the daily edition is available on android

Reply:
Yes, I will fix it in a new PR.